Makefile

```diff
@@ -1,7 +1,7 @@
 .PHONY: test
 test: pep8
 	export PYTHONPATH=$(pwd)/src;
-	cd tests \
+	export MEDIA_DIR=/tmp && cd tests \
```

Comment:
For the sake of consistency, shouldn't the `export` be on a separate line, just like the `PYTHONPATH` is?